docs/design/autoware-architecture/perception/index.md

```diff
@@ -1,5 +1,97 @@
 # Perception component design
 
-!!! warning
+## Purpose of this document
 
-    Under Construction
+This document outlines the high-level design strategies, goals and related rationales in the development of the Perception Component. Through this document, it is expected that all OSS developers will comprehend the design philosophy, goals and constraints under which the Perception Component is designed, and participate seamlessly in the development.
+
+## Overview
+
+The Perception Component receives inputs from Sensing, Localization, and Map components, and adds semantic information (e.g., Object Recognition, Obstacle Segmentation, Traffic Light Recognition, Occupancy Grid Map), which is then passed on to Planning Component. This component design follows the overarching philosophy of Autoware, defined as the [microautonomy concept](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-concepts/).
+
+## Goals and non-goals
+
+The role of the Perception component is to recognize the surrounding environment based on the data obtained through Sensing and acquire sufficient information (such as the presence of dynamic objects, stationary obstacles, blind spots, and traffic signal information) to enable autonomous driving.
+
+In our overall design, we emphasize the concept of [microautonomy architecture](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-concepts). This term refers to a design approach that focuses on the proper modularization of functions, clear definition of interfaces between these modules, and as a result, high expandability of the system. Given this context, the goal of the Perception component is set not to solve every conceivable complex use case (although we do aim to support basic ones), but rather to provide a platform that can be customized to the user's needs and can facilitate the development of additional features.
+
+To clarify the design concepts, the following points are listed as goals and non-goals.
+
+**Goals:**
+
+- The basic functions are provided so that a simple ODD can be defined.
+- To achieve a design that can provide Perception functionality to every autonomous vehicle.
+- The capability is extensible with the third-party components.
+- The Perception component is designed to provide a platform that enables autoware users to develop the complete functionality and capability.
```

Comment:
```suggestion
- The Perception Component is designed to provide a platform that enables autoware users to develop the complete functionality and capability.
```

(better to unify either to `The Perception Component` or `The Perception component`)